src/components/Icon/generated/DataCheckOutlined.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+const SvgDataCheckOutlined = (props: React.SVGProps<SVGSVGElement>) => (
```

Comment:
if you changed the svg file, change this one too
```suggestion
const SvgDataCheckOutline = (props: React.SVGProps<SVGSVGElement>) => (
```